src/components/ImageEditor.vue

```diff
@@ -207,7 +207,10 @@ export default {
 
 		// Key Handlers, override default Viewer arrow and escape key
 		handleKeydown(event) {
-			event.stopImmediatePropagation()
+			// Enter needs to be reached through as otherwise saving text does not work
+			if (event.key !== 'Enter') {
+				event.stopImmediatePropagation()
+			}
```

Comment:
@skjnldsv I need a bit of help here. This seems to fix the issue of not being able to save text with the enter key. However it also leads to changing the currently active picture. Can we somehow suppress modal key event listeners but not the event listener of the image editor?

Reply:
Maybe also @artonge has an idea?

Reply:
Is the image editor emitting an event when save the text ? If so, we might be able to abort there.

Also, I think it makes more sense to inhibit the viewer reaction to key pressed events from the viewer component and not the editor wrapper.

Reply:
> Is the image editor emitting an event when save the text ?

I am not sure. How can I find this out?

Reply:
Hm... doesnt look like this is the case.

> Also, I think it makes more sense to inhibit the viewer reaction to key pressed events from the viewer component and not the editor wrapper.

Ah, I see.

Reply:
Ah, I think this should be fixed with https://github.com/nextcloud/nextcloud-vue/commit/901d50940faab0001736d3363302aca097aaa098 in vue 7.6.0. So simply upgrading to 7.6.0 or higher should fix this behaviour.

Reply:
Indeed